### DISCUSSION TITLE
Re apply for Github Sponsor issue

### DISCUSSION THREAD

**Discussion by diamant3** [community/community#28178] [Other Feature Feedback, Questions, & Ideas]
Good day, I just wanna ask how to re apply my GitHub sponsor, I already submit a ticket to Github Support and please close with solution if this issue is already answered, thank you!

<img width="914" alt="error" src="https://user-images.githubusercontent.com/71203851/182610769-a3cf28f5-9edb-4df9-bba4-c71d61be0634.png">

**Comment by diamant3** [comment]
Good day, How can i enable my github sponsor account? thank you

![image](https://user-images.githubusercontent.com/71203851/182748804-93f75638-9331-4471-882d-530cca6a46eb.png)

**Reply by atompie** [reply]
I have the same issue. How can I reapply for the sponsor button? When I click the "You sponsors" I see this.
![Screenshot from 2022-08-05 23-07-08](https://user-images.githubusercontent.com/16271564/183184222-0531cc5d-2814-42ec-980e-c91d20f10db1.png)

**Reply by diamant3** [reply]
You can ask for support in [Github Support](https://support.github.com) and they fixed this issue but you will need to wait 2-3 days.

Right now i have an issue for stripe account so the sponsor button isn't alive right now on my profile and i'm working on it :)
